vendor/bat-native-ads/data/test/resources/nnqccijfhvzwyrxpxwjrpmynaiazctqb

```diff
@@ -0,0 +1,15 @@
+{
+  "locale": "us",
+  "version": 1,
+  "timestamp": "2021-04-26 09:00:00",
+  "conversion_id_patterns": {
+    "https://brave.com/foobar": {
+      "id_pattern": "<div.*id=\"conversion-id\".*>(.*)<\/div>",
```

Comment:
i think this needs a second backslash.
`"<div.*id=\"conversion-id\".*>(.*)<\\/div>"`